postgres_admin_panel/tasks/query_table_schema.sql

```diff
@@ -0,0 +1,8 @@
+SELECT 
+  table_name, 
+  column_name, 
+  data_type 
+FROM 
+  information_schema.columns 
+WHERE 
+  table_name = :table_name;
```

Comment:
pgformatter removes the extra spaces
```suggestion
SELECT
  table_name,
  column_name,
  data_type
FROM
  information_schema.columns
WHERE
  table_name = :table_name;

```